package/govuk-esm/vendor/polyfills/Date/now.mjs

```diff
@@ -1,12 +1,12 @@
 // @ts-nocheck
+
 (function (undefined) {
 
     // Detection from https://github.com/Financial-Times/polyfill-library/blob/v3.111.0/polyfills/Date/now/detect.js
```

Comment:
Do we also want to remove references to the FT polyfill repo? If you go to the repo it's been erased for containing malicious code. Github seems to be handling it but maybe good hygine? Whatcha reckon?

Reply:
Yeah, go on.